iotdb-core/ainode/ainode/core/ainode.py

```diff
@@ -159,6 +159,7 @@ def _handle_signal(self, signum, frame):
     def stop(self):
         if not self._stop_event.is_set():
             self._stop_event.set()
+            self._rpc_handler._inference_manager.shutdown()
```

Comment:
We'd better shutdown the `rpc_handler` rather than specific service belongs to the `rpc_handler`